Makefile

```diff
@@ -47,9 +62,12 @@ else
 	PLATFORMS := $(PLATFORM_LINUX_AMD_64),$(PLATFORM_LINUX_ARM_64)
 	WITH_MPI := 0
 	OFI_BUILD_ARG := WITH_OFI
+	NCCL_BUILD_ARG := WITH_NCCL
 	HOROVOD_WITH_MPI := 0
 	HOROVOD_WITHOUT_MPI := 1
 	HOROVOD_CPU_OPERATIONS := GLOO
+	HOROVOD_GPU_OPERATIONS := NCCL
+	HOROVOD_GPU_ALLREDUCE :=
```

Comment:
I am a bit worried this line turns off some optimizations for allreduce in the default images.
Also I don't see `HOROVOD_GPU_ALLREDUCE` being set to anything but empty anywhere. why is it necessary?

Reply:
Initially, I had them both set to NCCL, but got the following build error:
```
      CMake Error at cmake/Utilities.cmake:66 (message):
        Cannot specify both HOROVOD_GPU_OPERATIONS and HOROVOD_GPU_ALLREDUCE
        options.  Try unsetting one of these variables and reinstalling.
      Call Stack (most recent call first): 
```
I will delete the empty `HOROVOD_GPU_ALLREDUCE :=` line